applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -689,11 +681,12 @@ void UPwSmallStrainElement<TDim, TNumNodes>::CalculateOnIntegrationPoints(const
     rOutput.resize(number_of_integration_points);
 
     if (rVariable == CAUCHY_STRESS_VECTOR) {
-        for (unsigned int GPoint = 0; GPoint < mConstitutiveLawVector.size(); ++GPoint) {
-            if (rOutput[GPoint].size() != mStressVector[GPoint].size())
-                rOutput[GPoint].resize(mStressVector[GPoint].size(), false);
+        for (unsigned int integration_point = 0; integration_point < mConstitutiveLawVector.size();
+             ++integration_point) {
+            if (rOutput[integration_point].size() != mStressVector[integration_point].size())
+                rOutput[integration_point].resize(mStressVector[integration_point].size(), false);
```

Comment:
The `resize` operation implicitly performs this check; no need to repeat it here:
```suggestion
            rOutput[integration_point].resize(mStressVector[integration_point].size(), false);
```

Reply:
done